include/intersection.hpp

```diff
@@ -14,6 +14,7 @@ class intersection
 public:
   double target_density{};  // GeoDiv's target_density
   std::string geo_div_id;  // GeoDIv's ID
+  size_t pwh_idx;  // Polygon ID
```

Comment:
Is there any particular reason why are we using `size_t` instead of `unsigned int`?